conda/multisheller/ergocub-software_deactivate.msh

```diff
@@ -0,0 +1,5 @@
+if_(is_set("COMSPEC")).then_([
+	sys.list_remove("YARP_DATA_DIRS", path.join(env("CONDA_PREFIX"), "Library\\share\\iCub"))
+]).else_([
+	sys.list_remove("YARP_DATA_DIRS", path.join(env("CONDA_PREFIX"), "share/iCub"))
+])
```

Comment:
This does not seem to match the variables that are modified in `ergocub-software_activate.msh` . The basic ideas of this files is that you modify the env variables to undo the changes you do in `ergocub-software_activate.msh`.

Reply:
You are right, I forgot to save the file and I committed the one I copied from icub-models, it should be fine now